pinc/project_states.inc

```diff
@@ -397,5 +404,9 @@ function get_project_status_descriptor($which)
             die(html_safe("bad value for 'which': '$which'"));
     }
 
+    if ($testing) {
```

Comment:
This is fine for the PR, but this function could be used in places where we aren't outputting HTML (either because we're going to do `metaredirect()` or because we're using it in a script somewhere) so let's remove this before we merge it.

Reply:
Done this.